mediator/src/utils/fhir.ts

```diff
@@ -0,0 +1,75 @@
+export const VALID_GENDERS = ['male', 'female', 'other', 'unknown'] as const;
```

Comment:
That's FHIR patient resource gender type. Since we aren't preprocess the gender, it has to be on those predefined genders for incoming request https://build.fhir.org/patient.html